Mathlib/LinearAlgebra/Matrix/SpecialLinearGroup.lean

```diff
@@ -319,12 +330,70 @@ theorem fin_two_exists_eq_mk_of_apply_zero_one_eq_zero {R : Type*} [Field R] (g
   simp_rw [eq_inv_of_mul_eq_one_right had, hg]
 #align matrix.special_linear_group.fin_two_exists_eq_mk_of_apply_zero_one_eq_zero Matrix.SpecialLinearGroup.fin_two_exists_eq_mk_of_apply_zero_one_eq_zero
 
+lemma isCoprime_row (A : SL(2, R)) (i : Fin 2): IsCoprime (A i 0) (A i 1) := by
+    refine match i with
+    | 0 => ⟨A 1 1, -(A 1 0), ?_⟩
+    | 1 => ⟨-(A 0 1), A 0 0, ?_⟩ <;>
+    · simp_rw [det_coe A ▸ det_fin_two A.1]
+      ring
```

Comment:
```suggestion
  refine match i with
  | 0 => ⟨A 1 1, -(A 1 0), ?_⟩
  | 1 => ⟨-(A 0 1), A 0 0, ?_⟩ <;>
  · simp_rw [det_coe A ▸ det_fin_two A.1]
    ring
```

Reply:
Oops, this is my fault – looks like I'm getting my just desserts for telling off Chris about adherence for coding style rules